app/assets/stylesheets/application.css.scss

```diff
@@ -13,3 +13,9 @@
  *= require_tree .
  *= require_self
  */
+@import "bootstrap-sprockets";
+@import "bootstrap";
+@import "bootstrap/theme";
+body{
+  margin-top: 60px;
+}
```

Comment:
As the above commits about front-end library being not related to this feature PR, this should not be committed also. And we should not have any css in the application.css, this should be moved to other css file and imported in the application.css. Ideally our application.css should only have imports.